src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateRepresentedClaimCopyUsingPdfServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.cmc.claimstore.BaseIntegrationTest;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleAmountRange;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.reform.cmc.pdf.service.client.exception.PDFServiceClientException;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class GenerateRepresentedClaimCopyUsingPdfServiceTest extends BaseIntegrationTest {
```

Comment:
Can we just rename it to be `GenerateRepresentedClaimCopyTest`. This test is just about generating PDF. On top of that we will have `RetrieveRepresentedClaimCopyUsingDocumementManagementTest` which will cover only DM integration.

Reply:
done